module-services/service-evtmgr/battery/BatteryController.hpp

```diff
@@ -20,7 +20,9 @@ namespace sevm::battery
       public:
         using Events          = hal::battery::AbstractBatteryCharger::Events;
         using ChargerPresence = hal::battery::AbstractBatteryCharger::ChargerPresence;
-        explicit BatteryController(sys::Service *service, xQueueHandle notificationChannel);
+        explicit BatteryController(sys::Service *service,
```

Comment:
Remove `explicit`. It is only required if constructor takes one argument :)

Reply:
done